e2e/page-objects/header.po.ts

```diff
@@ -10,16 +10,16 @@ export class HeaderPage {
   userMenu = element(by.css('.navbar-user'));
   logout = element(by.css('.logout'));
 
-  getElement (name) {
+  getElement (name, getChild) {
```

Comment:
Looks good.  Didn't know you could add `.child` to a protractor element like that